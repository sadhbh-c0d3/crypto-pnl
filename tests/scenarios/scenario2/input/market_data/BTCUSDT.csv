1654077300000,25000.00000000,25000.00000000,25000.00000000,25000.00000000,,,,,,
1654077600000,25000.00000000,25000.00000000,25000.00000000,25000.00000000,,,,,,
1654077900000,25000.00000000,25000.00000000,25000.00000000,25000.00000000,,,,,,
1654078200000,25000.00000000,25000.00000000,25000.00000000,25000.00000000,,,,,,