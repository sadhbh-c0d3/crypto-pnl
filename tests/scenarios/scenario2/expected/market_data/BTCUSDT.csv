1654077900000,25000.00000000,25000.00000000,25000.00000000,25000.00000000
